inst/WORDLIST

```diff
@@ -43,6 +43,7 @@ BDS
 BILI
 BLQ
 BMI
+BMIBL
```

Comment:
I think this is not required with the suggested change in the `derive_vars_computed()` documentation.